tasks/test.rake

```diff
@@ -7,7 +7,6 @@ task :setup, :parallel do |t, opts|
     system 'mkdir -p spec/rails'
     args = %w[
       -m\ spec/support/rails_template.rb
```

Comment:
Why? This is used for tests only, I think it's better like it is now... without data. I don't think the tests would even pass if we seed the test DB.